providers/namedotcom/namedotcomProvider.go

```diff
@@ -19,6 +19,13 @@ type nameDotCom struct {
 	APIKey  string `json:"apikey"`
 }
 
+var docNotes = providers.DocumentationNotes{
+	providers.DocDualHost:            providers.Cannot("Apex NS records not editable"),
```

Comment:
Apex NS records are editable, but they have to match the list of nameservers at the registry.

Reply:
That's interesting @pmoroney. It creates a bit of a chicken and egg problem. In dnscontrol we update the dns provider first, and then the registrar. It sounds like that wouldn't work with NDC unless we invert that order? 

We made that choice specifically so we never point at an uninitialized dns server.

Reply:
The dns provider ignores the Apex NS records and just lets the registrar provider handle it, so it doesn't really matter which order it works in. I do it for pat.irish using dnscontrol. I just removed it and re-added it and didn't have any problems.

Reply:
Oh, that makes sense. Thats a slightly different paradigm than other providers, but it should work fine. I'll run a quick test and update the flag.